test/integration.mjs

```diff
@@ -228,7 +227,13 @@ function testCmd(cmd, args, cb) {
   const cmds = cmd.split(' ');
 
   const bin = spawn(cmds[0], cmds.slice(1).concat(args.map(String)), {
-    cwd: path.join(__dirname, '/fixtures'),
+    cwd: path.resolve('./test/fixtures'),
+    env: {
+      ...process.env,
+      ...{
+        NODE_OPTIONS: '--experimental-require-module',
```

Comment:
Is this ENV still needed, or perhaps left over from earlier work before enforcing node version?

Reply:
I believe this is no longer needed. Removed.